packages/x-date-pickers/src/tests/describeGregorianAdapter/testCalculations.ts

```diff
@@ -296,7 +296,9 @@ export const testCalculations: DescribeGregorianAdapterTestSuite = ({
     expect(adapter.getDiff(testDate, adapter.date('2018-10-31T11:44:00.000Z')!)).to.equal(
       -86400000,
     );
-    expect(adapter.getDiff(testDate, '2018-10-31T11:44:00.000Z')).to.equal(-86400000);
+    expect(adapter.getDiff(testDate, adapter.date('2018-10-31T11:44:00.000Z')!)).to.equal(
+      -86400000,
+    );
```

Comment:
Am I just blind or is there no difference between the above assertion and this one? 🤔 🤷 
```suggestion
```

Reply:
The date was not parsed, probably by mistake, you have an additional `adapter.date` now